.github/CODEOWNERS

```diff
@@ -19,8 +19,8 @@
 /jekyll/ja/ @circleci/translations
 *_ja* @circleci/translations
 
-/src/** @circleci/docs-disco
-*package*.json @circleci/docs-disco
-*.lock @circleci/docs-disco
-Gemfile* @circleci/docs-disco
-.*rc* @circleci/docs-disco
+/src/** @circleci/cpeng
```

Comment:
![Screen Shot 2023-01-12 at 9 17 48 AM](https://user-images.githubusercontent.com/1449325/212135312-7b9030e9-6b05-4394-a3fd-34389dd24f30.png)

Is this intended?